### ISSUE TITLE
Design Spec

### ISSUE THREAD

**Issue by josephgupkim** [Coralie9/f1-3-c2p1-colmar-academy#3]
Your project from a laptop/non-mobile standpoint looks nearly identical to the design spec which means you were able to apply the desired styling to the necessary elements.

The mobile view seems a bit lacking and it could use a bit more work.

- The pictures come out stretched.
- The Thesis Exhibit video should stretch across the screen.
- The nav bar seems to overlap the banner.

**Comment by Coralie9** [comment]
Yes I was able to apply the desired styling to the necessary elements but it is true that the mobile view kind of gave me a hard time. For the pictures stretching and the nav bar overlapping the banner, I will try to find a solution. But for the video, I tried but I could not manage to stretch it across the screen. Can you please help me?
